### DISCUSSION TITLE
ERROR: AlreadyExists: ... (the error 183) to book project on google drive folder

### DISCUSSION THREAD

**Discussion by jefferds** [quarto-dev/quarto-cli#2175] [Q&A]
when I use a folder on google drive for the book projects I get the following: ERROR: AlreadyExists: ... (the error 183) ... mkdir 'H:\My Drive\001-Projetos\...\site_libs\quarto -nav'. Does anybody know how to solve this?

**Comment by mcanouil** [comment]
You have to give more if you want an actual answer.
Here, you don't provide anything beside an error, no context, no code.
Please do provide some context. 😉

**Reply by jefferds** [reply]
Sorry, I'm not sure how to describe it, it's my first time here. 

When I create a book project and use google drive to sync my projects between the computers I work on I get ERROR 183 every time I try to render a new page. Journal Articles type projects do not show the error.

**Reply by mcanouil** [reply]
Basically, what Quarto version are you using? Are you using Quarto CLI directly or through RStudio, the quarto R package, the Python library, etc.
Describe what you did, such as the commands/functions you used, the tools you used, etc.
The idea is to allow others to reproduce your error.

**Reply by jefferds** [reply]
Google Drive - v63.0.5.0
Quarto - v1.1.189-win
VSCode - v1.71.0
Quarto extension - v1.38.0

I followed the steps:
`quarto create-project mybook --type book`
`quarto preview`

The book is created and opens normally, but when I try to navigate to *Introduction* or *Summary* I get the error.  The error only occurs if the book is created in a synced Google Drive folder.

**Reply by jefferds** [reply]
Watching files for changes
Browse at http://localhost:5239/
GET: /
ERROR: AlreadyExists: Não é possível criar um arquivo já existente. (os error 183), mkdir 'H:\Meu Drive\mybook\site_libs\quarto-nav'
GET: /intro.html
GET: /
ERROR: AlreadyExists: Não é possível criar um arquivo já existente. (os error 183), mkdir 'H:\Meu Drive\mybook\site_libs\quarto-nav'
GET: /intro.html
GET: /
GET: /
ERROR: AlreadyExists: Não é possível criar um arquivo já existente. (os error 183), mkdir 'H:\Meu Drive\mybook\site_libs\quarto-nav'
GET: /summary.html
ERROR: AlreadyExists: Não é possível criar um arquivo já existente. (os error 183), mkdir 'H:\Meu Drive\mybook\index_files\mediabag'
GET: /

**Reply by mcanouil** [reply]
Could you post that on the opened issue? #2174

Also, Do you have the same error in a directory not in Google Drive (or any other cloud framework)? See #2395 for other related Google Drive issues.

@dragonstyle I am pinging you because of #2395 since #2174 might be related. (Maybe closing the discussion to continue on the issue).

**Reply by cderv** [reply]
Thanks @mcanouil, let's lock the thread here and continue in the issue. Thanks !

**Comment by cderv** [comment]
Related issue on this https://github.com/quarto-dev/quarto-cli/issues/2174
